src/rshim_pcie_lf.c

```diff
@@ -186,6 +186,8 @@ static int msn_gw_read(struct pci_dev *pci_dev, int addr,
   uint32_t data;
   int rc = 0;
 
+  addr &= ~BF3_RSH_ADDR_MASK;
```

Comment:
This bit-clear of an address looks having a specific purpose. Better to add a comment here.

Reply:
done